lib/assets/it.jst

```diff
@@ -10,12 +10,14 @@ if (saveHtml) {
 }
 %>
 it('should be equal `${ it }` by ${ engine.name }${ titleSuffix }', function (<% if (saveHtml || engine.async) { %>done<% } %>) {
+    ${ before }
+
     var bemjson = references['${ it }']${ subreference }.bemjson,
         expected = references['${ it }']${ subreference }.html;<%
 
 if (!engine.async) { %>
     // sync mode
-    var actual = engines['${ engine.name }']${ subreference }.apply(bemjson);
+    var actual = engine.apply(bemjson);
```

Comment:
Let's apply this to async mode as well: https://github.com/enb/enb-bem-tmpl-specs/pull/125/files#diff-b3ae0a4158614ff6d4c87ca1707c1221R30